src/Task_2/Month.java

```diff
@@ -0,0 +1,57 @@
+package Task_2;
+
+import java.util.Scanner;
+
+public class Month {
+
+    public static void main(String args[]){
+
+        Scanner reader = new Scanner(System.in);
+        System.out.println("Enter number: ");
+        int n = reader.nextInt();
+
+        switch (n) {
```

Comment:
can you think of a way to avoid switch-case (or if-else) to solve this problem?

Reply:
System.out.println("Enter number from 1 to 12: ");
int n = reader.nextInt();

String[] month = {"Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"};
System.out.println = (month[n-1]);

Reply:
you can add a separate method with this task solution